presto-common/src/main/java/com/facebook/presto/common/RuntimeStats.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.common;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Metrics exposed by presto operators or connectors. These will be aggregated at the query level.
+ */
+public class RuntimeStats
+{
+    private final Map<String, RuntimeMetric> metrics;
+
+    public RuntimeStats()
+    {
+        metrics = new HashMap<>();
```

Comment:
nit: I think we could just put this in the declaration part? 

`private final Map<String, RuntimeMetric> metrics = new HashMap<>();`

Reply:
There is another JsonCreator constructor below, in which case we don't want to create a new map by default.

Reply:
I see, good catch I overlooked. Maybe we could do `this(new HashMap<>())` instead?

Reply:
I see, good catch I overlooked. Maybe we could do `this(new HashMap<>())` instead?